bin/daemon.php

```diff
@@ -62,41 +63,45 @@
 	die("Unexpected script behaviour. This message should never occur.\n");
 }
 
-$pid = @file_get_contents($pidfile);
+$id = @file_get_contents($pidfile);
```

Comment:
`$pid` was a more precise name. Can you please revert this back?

Reply:
No, there's a `process` table with an `id` AND a `pid` field. The `pid` used to be the primary key but it isn't sufficient for multi-host Friendica installations. Hence the add of the more generic `id` primary key field.